### ISSUE TITLE
Add code for circular Linked List in Linked List folder

### ISSUE THREAD

**Issue by rajkumawat01** [PCCoE-Hacktoberfest-21/Data-Structures-and-Algorithms#26]

**Comment by rajkumawat01** [comment]
Can you assign me this issue?

**Comment by rajkumawat01** [comment]
in cpp

**Comment by SumedhaZaware** [comment]
@rajkumawat01 yes, go for it!!

**Comment by Rohit-More** [comment]
@SumedhaZaware Can you assign this issue to me as @rajkumawat01 is already working on several issues?

**Comment by SumedhaZaware** [comment]
@Rohit-More It won't be fair on his part. If you want you can work on the same issue using any other language except CPP.

**Comment by OmkarBansode07** [comment]
Can you assign this issue to me??

**Comment by SumedhaZaware** [comment]
@OmkarBansode07 Do you want to do this in cpp?

**Comment by OmkarBansode07** [comment]
> @OmkarBansode07 Do you want to do this in cpp?
Yes, I wan to do in CPP.

**Comment by OmkarBansode07** [comment]
> @OmkarBansode07 Do you want to do this in cpp?

@SumedhaZaware  Yes I want to do in CPP

**Comment by SumedhaZaware** [comment]
@OmkarBansode07 It is already assigned to @rajkumawat01, If you want you can do it in some other language

**Comment by OmkarBansode07** [comment]
@SumedhaZaware Yes, I can do it in another language .So you can assign this issue to me

**Comment by Vedantnerkar** [comment]
@OmkarBansode07 go ahead

**Comment by OmkarBansode07** [comment]
I am doing in C language

**Comment by Rohit-More** [comment]
@SumedhaZaware  can you assign this issue to me? I would be using Java to solve it.

**Comment by adarshcode** [comment]
Hi @SumedhaZaware . Can i contirbute to this issue in C?

**Comment by moonlight-blue** [comment]
Can I work on this issue using Python?
